src/main/java/com/cloudbees/hudson/plugins/folder/computed/DefaultOrphanedItemStrategy.java

```diff
@@ -230,6 +269,27 @@ private static boolean disabled(TopLevelItem item) {
 
     @Override
     public <I extends TopLevelItem> Collection<I> orphanedItems(ComputedFolder<I> owner, Collection<I> orphaned, TaskListener listener) throws IOException, InterruptedException {
+        if (abortBuilds) {
+            Queue<Run<?, ?>> abortedBuilds = new LinkedList<>();
+            for (I item: orphaned) {
+                for (Job<?, ?> job: item.getAllJobs()) {
+                    for (Run<?, ?> build: job.getBuilds()) {
+                        Executor executor = build.getExecutor();
```

Comment:
I believe this will work for Pipeline builds, but indeed it would be nice to have test coverage proving it.

Would I think need a `test`-scoped dep on `workflow-multibranch` and the `tests` classifier of `scm-api` to make a multibranch project governed by a mock branch source that you can manipulate.